doc/source/dev/internals.code-explanations.rst

```diff
@@ -563,10 +563,7 @@ way, the operation performed is
 .. math::
    :nowrap:
 
-   \begin{align*}
-   o & = & i[0] \\
-   o & = & i[k]\textrm{<op>}o\quad k=1\ldots N
-   \end{align*}
+   $$o_k = \begin{cases} & i[0] & k=0 \\\ & i[k] \text{ <op> }o_{k-1} & k=1,\ldots, N \text{.} \end{cases}$$
```

Comment:
Why did you change `o` to be a vector? It is a single value in a reduce. That is the point of reduction. It is initialized to the value of `i[0]` and then subsequent iterations update it `o = i[k] <op> o`.